core/storage/user/gae_models.py

```diff
@@ -110,6 +110,22 @@ class UserSettingsModel(base_models.BaseModel):
     preferred_audio_language_code = ndb.StringProperty(
         default=None, choices=[
             language['id'] for language in constants.SUPPORTED_AUDIO_LANGUAGES])
+    # Displayed name for users on android. They are editable unlike usernames.
+    name = ndb.StringProperty(default=None)
+    # A code associated with profiles to provide a PIN based authentication
+    # within the account on android.
+    pin = ndb.StringProperty(default=None)
+    # List of profile user ids associated with a full user. None for profiles.
+    associated_profile_user_ids = ndb.StringProperty(repeated=True)
+    # Exploration language preferences specified by the user for android.
+    preferred_android_language_codes = ndb.StringProperty(
```

Comment:
I thought I asked this before but why would these be different from preferred_language_codes, etc. above? The comments should explain.

Reply:
Explained in the comments now. Also pls refer to the discussion going on in above [comment thread](https://github.com/oppia/oppia/pull/10105#discussion_r462701426).